### DISCUSSION TITLE
How to embed forms developed in formsflow.ai to other applications

### DISCUSSION THREAD

**Discussion by sumesh-aot** [AOT-Technologies/forms-flow-ai#1190] [Q&A] [answered]
Customers often have requirement to embed the forms developed in formsflow.ai into other applications which are using different tech stack and framework. Does formsflow.ai support this ?

**Comment by AbijithS-aot** [comment, marked as answer]
> Customers often have requirement to embed the forms developed in formsflow.ai into other applications which are using different tech stack and framework. Does formsflow.ai support this ?

Yes formsflow.ai supports form embedding, you can embed the form which is created in formsflow.ai to any websites.
As of now formsflow.ai supports three types of form embedding 
 *  Anonymous forms
 where you can embed your anonymous forms.
 *  Authenticated(Internal)
  If your parent application uses keycloack for authentication then, by passing keycloak url , realm  and client Id you can embed your form in parent application.
 *  Authenticated(external)
 If your parent application is not using keycloak for authentication, you can pass a jwt token which is encoded with the shared secret of formsflow.ai and you can embed the form

you can refer the  link for more details
[https://github.com/AOT-Technologies/forms-flow-ai-web-components](https://github.com/AOT-Technologies/forms-flow-ai-web-components)
